example-app/src/main/java/com/adyen/checkout/example/CheckoutExampleApplication.kt

```diff
@@ -8,13 +8,13 @@
 
 package com.adyen.checkout.example
 
+import android.app.Application
 import android.util.Log
-import androidx.multidex.MultiDexApplication
 import com.adyen.checkout.core.log.Logger
 import dagger.hilt.android.HiltAndroidApp
 
 @HiltAndroidApp
-class CheckoutExampleApplication : MultiDexApplication() {
```

Comment:
Can you link me some info on why this is not relevant anymore? I'm a bit outdated I guess.

Reply:
https://developer.android.com/studio/build/multidex#mdex-on-l